src/control/cmd/ddb/ddb_types.go

```diff
@@ -0,0 +1,18 @@
+package main
+
+type CommandContext struct {
+	CContext          *DdbContext
+	jsonOutput        bool
+	jsonOutputHandled bool
+}
+
+type SuperBlock struct {
+	PoolUuid             string `json:"uuid"`
```

Comment:
It might be better to have this field as a `uuid.UUID` instead of a string. It might also be useful to move this struct to the `lib/daos` package and rename it to `PoolSuperblock`, not sure.

Reply:
Will update to uuid.UUID. 

There will be several other ddb types added as I convert more ddb to go. Would it make sense to include them all in lib/daos? Other structures would be like DTX, ILOG, and VEA entries.

Reply:
lib/daos feels like the right place for these to me.